### DISCUSSION TITLE
Targeting "spacer" div

### DISCUSSION THREAD

**Discussion by mattevans-dev** [muxinc/elements#363] [General]
Hi,
Thanks for a great library, really enjoying using it.
Just wondering if it's possible to target the "spacer" class. I've added a flex-grow to the display time as I want to shift the volume controls to the end. However, I've noticed there is a div called spacer which is blocking what I am trying to achieve. I'm trying to hide it but I'm struggling to target it through CSS. Any suggestions?

<img width="1306" alt="image" src="https://user-images.githubusercontent.com/55884592/188474563-94732e8e-2831-40aa-9db7-4edc60295560.png">

Thanks,
Matt

**Comment by luwes** [comment]
@mattevans-dev thanks for the question / feedback. 
At the moment it's not possible to target the spacer with CSS vars or parts but I think it should be. We'll take a look at this soon.

**Reply by TSIA-SN** [reply]
Was this ever done? I see that the spacer has a class of 'spacer' and that the color looks like it could potentially be changed to some extent with the `--media-control-background` variable, but as it doesn't have a part set on it, it seems like it couldn't be fully customized.
